packages/good-design/src/core/buttons/LearnButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from "react";
+import { Box, Text } from "native-base";
+import BasePressable from "./BasePressable";
+import { openLink } from "@gooddollar/web3sdk-v2";
+import { Image } from "../images";
+import BackToSchool from "../../assets/images/backtoschool.png";
+
+type learnLinks = {
+  transactions: string;
+  bridging: string;
+  network: string;
+  signing: string;
+};
+const links: learnLinks = {
```

Comment:
why not compose onto single map with { link: string; label: string } value ?

e.g. 

```typescript
const links: Record<string, { link: string; label: string }> = {
  transactions: {
    label: "How do transactions work?",
    link: "https://www.notion.so/gooddollar/How-do-transactions-work-in-web3-ccf11b4e16874a1682722a20c4e24742"
  },
  // ...
}
```

Reply:
done